news/1 Enhancements/6025.md

```diff
@@ -0,0 +1 @@
+Replace 'Run cells above' and 'Run cell and below' commands and cell toolbar buttons with VS Code's built-in 'Execute Above Cells' and 'Execute Cell And Below' commands and unified run button.
```

Comment:
I'm wondering if 'interrupt' is going to work correctly or not. We used to be able to cancel all the queued cells for run all when interrrupting. I'm guessing that doesn't work anymore.

Reply:
We probably will need some special logic in the executeCell commands to handle it.